tree/ntuple/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -1141,6 +1194,59 @@ public:
    RIterator end() { return RIterator(fNTuple.fExtraTypeInfoDescriptors.cend()); }
 };
 
+namespace Experimental {
+// clang-format off
+/**
+\class ROOT::Experimental::RNTupleAttrSetDescriptorIterable
+\ingroup NTuple
+\brief Used to loop over all the Attribute Sets linked to an RNTuple
+*/
+// clang-format on
+// TODO: move this to RNTupleDescriptor::RNTupleAttrSetDescriptorIterable when it moves out of Experimental.
+class RNTupleAttrSetDescriptorIterable final {
+private:
+   /// The associated RNTuple for this range.
+   const RNTupleDescriptor &fNTuple;
+
+public:
+   class RIterator final {
+   private:
+      using Iter_t = std::vector<RNTupleAttrSetDescriptor>::const_iterator;
+      /// The wrapped map iterator
+      Iter_t fIter;
+
+   public:
+      using iterator_category = std::bidirectional_iterator_tag;
```

Comment:
The tag is not correct, the iterator doesn't implement enough methods. I think we recently concluded that even for iterators wrapping `std::vector`, we only want to expose `std::forward_iterator_tag` (see `RExtraTypeInfoDescriptorIterable`)